app/javascript/selectors/detailSelector.js

```diff
@@ -189,5 +189,5 @@ export const phoneExtension = state => {
 
 export const userNotes = state => {
   const notes = safeGet(state, 'fetchDetails.details.records.user.notes') || ''
-  return { notes: notes, notesLength: notes.length }
+  return { notes: notes, notesLength: notes.normalize().length }
```

Comment:
normalize seems not to do anything for us.  See if you get the same functionality if you get rid of it everywhere.
```suggestion
  return { notes: notes, notesLength: notes.length }
```